addons/mail/static/src/core/common/chat_window_container.xml

```diff
@@ -6,7 +6,7 @@
         <div t-if="chatWindowService.hidden.length > 0 and !ui.isSmall" t-ref="hiddenMenu" class="o-mail-ChatWindow-hiddenMenuContainer position-fixed bottom-0">
             <t t-call="mail.ChatWindowHiddenMenu"/>
         </div>
-        <t t-foreach="chatWindowService.visible" t-as="chatWindow" t-key="chatWindow.thread?.localId">
+        <t t-foreach="[...chatWindowService.visible]" t-as="chatWindow" t-key="chatWindow.thread?.localId">
```

Comment:
cf comment in master, this is not a good sign, code should be improved to make that change not necessary